lms/urls.py

```diff
@@ -91,6 +91,8 @@
     # independently of courseware. https://github.com/edx/edx-val
     url(r'^api/val/v0/', include('edxval.urls')),
 
+    # Full Course/Library Import/Export API
+    url(r'^api/import_export/v1/', include('openedx.core.djangoapps.import_export.urls')),
```

Comment:
So that the API is accessible from both LMS and CMS. That was one of the things I thought would be controversial, but I also don't think this is not strictly necessary. It just makes us able to export courses using this API residentially if we don't want to run studio in production. (Thoughts on this @carsongee ?) For imports/exports with edx.org it isn't necessary.